packages/SketchMorph2-Core.package/MorphicMonet.class/instance/chooseColor.st

```diff
@@ -0,0 +1,9 @@
+tooling
+chooseColor
+	| currentColor |
+	self world activeHand showTemporaryCursor: 
+		(CursorWithAlpha fromIcon: UiFugueIcons pipetteIcon).
```

Comment:
I don't think we should use `UiFugueIcons` since we cannot ensure the package will be installed.